tools/buildInstaller.js

```diff
@@ -17,7 +17,41 @@ if (isWindows) {
     widevineCdmArch = 'win_x86'
   }
 }
-const buildDir = 'Brave-' + process.platform + '-' + arch
+
+const channel = process.env.CHANNEL
+
+var channels = { nightly: true, developer: true, beta: true, dev: true }
+if (!channels[channel]) {
```

Comment:
This could be changed to import the channels directly from `app/channel.js`. 
`const channels = require('../app/channel').channels`

If that is an array:
`if (!channels.includes(channel)) {`

if that is a Set:
``if (!channels.has(channel)) {`

Reply:
It seems weird to me to pull dependency from `app` folder for `tools`

Reply:
yeah that's true- maybe it's fine as-is 😄